docs/docs/reference/cloud/monorepos.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Monorepos"
+description: "Monorepos support in Gatsby Cloud"
+---
+
+## Introduction
+
+Gatsby Cloud has support for monorepos projects, but since there is a plethora of dependency management tools out there some issues can occure.
+This document will walk you through the tools that are supported and how to troubleshoot some edge cases
```

Comment:
```suggestion
Gatsby Cloud has support for projects organized as [monorepos](https://monorepo.tools/), but since there are different competing tools to build a monorepo it's important to know what Gatsby Cloud supports today.

This document will explain which tools are supported and how you can troubleshoot issues.
```